### PR TITLE
List (mobile): Fix to apply a GUI feedback part.2

### PR DESCRIPTION
**Pull request by HunseopJeong**
[Issue] https://github.com/Samsung/TAU/issues/980
[Problem] Need to adjust some styles according to GUI feedback
[Solution]
- Adjust the basic list 1, 2, and 3 line
- Add some icons and remove an unnecessary margin

Signed-off-by: Hunseop Jeong <hs85.jeong@samsung.com>

### CODE REVIEW

File: src/css/profile/mobile/common/listview.less [hunk 1]
```diff
@@ -140,25 +140,34 @@ tau-expandable {
...
+			padding: 15 * @px_base 0;
 			margin-right: 24 * @px_base;
 
 			.ui-li-text-title {
 				font-size: 18 * @sp_base;
 				color: var(--text-color);
+				line-height: 21 * @px_base;
+
+				&:only-child {
+					padding-bottom: @px_base;
```
Comment (DohyungLim) [inline src/css/profile/mobile/common/listview.less:156]: There is no value. Is it only intended to fill in the units?

**Review by DohyungLim — COMMENTED** [review]
1 comment.

File: src/css/profile/mobile/common/listview.less (hunk 1 above)
Comment (HunseopJeong) [reply src/css/profile/mobile/common/listview.less:156]: @px_base == 1px

File: examples/mobile/UIComponents/components/list/icon-text-list.html [hunk 2]
```diff
@@ -26,7 +26,7 @@
 			<ul class="ui-listview ui-content-area">
 				<li class="ui-li-has-icon ui-li-divider">
 					<div class="ui-li-icon">
-						<img src="images/thumbnail.jpg"/>
+						<img src="images/tw_list_icon_connections.svg"/>
```
Comment (singa2000) [inline examples/mobile/UIComponents/components/list/icon-text-list.html:29]: In the Guide, the icons also have the each colors like "#53a4ff" etc.
Can you apply the color?

File: src/css/profile/mobile/common/listview.less [hunk 3]
```diff
@@ -140,25 +140,34 @@ tau-expandable {
 			}
 		}
 
-		&.ui-li-has-icon {
```
Comment (singa2000) [inline src/css/profile/mobile/common/listview.less:143]: Is it really  delete the ui-li-has-icon left margin??
Is it correct?

File: examples/mobile/UIComponents/components/list/icon-text-list.html (hunk 2 above)
Comment (HunseopJeong) [reply examples/mobile/UIComponents/components/list/icon-text-list.html:29]: I applied a color directly to the svg file.
```
<svg .... fill="#53a4ff">
```

File: src/css/profile/mobile/common/listview.less (hunk 3 above)
Comment (HunseopJeong) [reply src/css/profile/mobile/common/listview.less:143]: Yes, it is.
Icon already has a margin value.
```
margin: 0 12 * @px_base;
```

**Review by DohyungLim — APPROVED** [review]
LGTM.

**Review by singa2000 — APPROVED** [review]
LGTM

**Pull request merged**